### PR TITLE
#165413075 user can follow and unfollow each other

### PR DESCRIPTION
**Pull request by Kabalisa**
Description
=======
This feature is about allowing an authenticated user to follow an author on authors haven.

Type of change
=======
N/A

How has it been tested
=======
- pull this branch
- run ```npm install```
- run ```npm start``` to start our react app
-login into authorshaven
-choose an article written by another author
-follow the author
- unfollow the author
-navigate to your profile
- Click on either the following or followers header
- a page showing your followers and who you follow will appear
- there you can follow and unfollow authors.

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#165413075](https://www.pivotaltracker.com/story/show/165413075)

### CODE REVIEW

File: src/components/follow/follow.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                    <div>{Following.username}</div>
+                    <Button
+                      value='following'
+                      className='following on-profile'
+                      onClick={() => this.handleFollow(Following.username)}
+                    />
+                  </div>
+                ))
+              ) : (
+                <div>you do not follow anyone</div>
```
Comment (AJAkimana) [inline src/components/follow/follow.jsx]: Start with a capital letter sentences

File: src/components/follow/follow.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                      <Button
+                        value='follow'
+                        className='follow on-profile'
+                        onClick={() => this.props.followUser(follower.username)}
+                      />
+                    )}
+                  </div>
+                ))
+              ) : (
+                <div>you have no followers</div>
```
Comment (AJAkimana) [inline src/components/follow/follow.jsx]: Start with a capital letter

File: src/components/follow/follow.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        state
+      });
+    }
+  }
+  render() {
+    const { following, followers } = this.props.location.state;
+    return (
+      <div>
+        <NavBar />
+        <div className='Follow'>
```
Comment (AJAkimana) [inline src/components/follow/follow.jsx]: Put a title on this page that indicates users where they are

**Review by AJAkimana — COMMENTED** [review]
Good work @Kabalisa, Consider the inline comments

File: src/components/follow/follow.jsx (hunk 3 above)
Comment (Kabalisa) [reply src/components/follow/follow.jsx]: @AJAkimana. the titles are already there. there is one above followers and another above following.

**Review by jabichris — APPROVED** [review]
Nice work

**Review by AJAkimana — APPROVED** [review]
Good work @Kabalisa

**Pull request merged**